app/controllers/additional_document_validation_requests_controller.rb

```diff
@@ -8,8 +8,8 @@ def create
     @additional_document_validation_request.user = current_user
 
     if @additional_document_validation_request.save
-      flash[:notice] = "Document create request successfully sent."
-      send_validation_request_email
+      flash[:notice] = "Document create request successfully created."
```

Comment:
maybe 
```
Document create request successful
```
would probably avoid repeated the "create", just an idea

Reply:
Have updated - thanks!